dex/runner.go

```diff
@@ -78,23 +78,23 @@ type Connector interface {
 	Connect(ctx context.Context) (error, *sync.WaitGroup)
 }
 
-// ConnectionMaster manages a Connector.
-type ConnectionMaster struct {
+// ConnectionManager manages a Connector.
+type ConnectionManager struct {
```

Comment:
I usually reserve `Manager` for types that handles multiple things, not just one. What is the reasoning for the name change?

Reply:
It appear the type simply wraps a connector for the purpose of managing its lifecycle with the provided `Connect` and `Disconnect` methods. Could use a different name but `Master` had me wondering for a while what the type really does.

Reply:
`Manager` is not an improvement. Please don't change type names.